### DISCUSSION TITLE
Remote covers in single-user instance

### DISCUSSION THREAD

**Discussion by dato** [bookwyrm-social/bookwyrm#2997] [Q&A]
I'm trying BookWyrm in a single-user instance (with the hope this might make it easier to contribute one day), and I'm a bit saddened by my feed because no book has a cover.

I see that book description and other metadata are copied over when importing the book into my instance. Is this supposed to happens for covers too (and in that case there's an issue with my setup), or not done for some particular reason?

At first I thought it had to do with "link domains", but it doesn't seem to be the case. Any ideas?

Many thanks in advance!

**Comment by prolibre** [comment]
Hello,

I'm a new instance administrator, so my answer will be incomplete. I've noticed that book covers often arrive later when you import them from another bookwyrm instance (probably with a background task). But there are still things I don't understand: sometimes the cover is not downloaded from instance to instance (even though everything seems normal).

If you're talking about importing from a site (Open Library for example) I think that works relatively well.
